docs/running-on-kubernetes.md

```diff
@@ -932,15 +935,31 @@ specific to Spark on Kubernetes.
   <td>(none)</td>
   <td>
    Specify the local file that contains the driver [pod template](#pod-template). For example
-   <code>spark.kubernetes.driver.podTemplateFile=/path/to/driver-pod-template.yaml`</code>
+   <code>spark.kubernetes.driver.podTemplateFile=/path/to/driver-pod-template.yaml</code>
+  </td>
+</tr>
+<tr>
+  <td><code>spark.kubernetes.driver.podTemplateContainerName</code></td>
+  <td>(none)</td>
+  <td>
+   Specify the container name to be used as a basis for the driver in the given [pod template](#pod-template).
```

Comment:
Actually, `[pod template](#pod-template)` is not working. Could you try the following and check with `SKIP_API=1 jekyll build`?
```
<a href="#pod-template">pod template</a>
```